salt/metalk8s/addons/volumes/volume-crd.sls

```diff
@@ -0,0 +1,77 @@
+#!jinja | kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
```

Comment:
Hmh, could this be generated from the file as generated by `operator-sdk generate ...` (i.e., the file in `storage-operator/disk/...`) at build time, instead of copying it over?

Reply:
Yup I plan to do it in a later commit (PR still in draft ^^), first I try to make it work like that, then I'll make sure we reuse the one generated by the SDK to avoid desync issue.

Reply:
Done